packages/wct-mocha/tsconfig.json

```diff
@@ -11,20 +11,21 @@
     "strictNullChecks": false,
     "removeComments": false,
     "preserveConstEnums": true,
+    "skipDefaultLibCheck": true,
+    "skipLibCheck": true,
```

Comment:
this has a lot of unstrict things, can they be removed?

Reply:
Okay I was able to remove these after adjusting declarations file a little.  Couldn't get rid of all the unstrict stuff tho
-    "strictNullChecks": false,
-    "skipDefaultLibCheck": true,
-    "skipLibCheck": true,

Reply:
I think these are legitimate issues, though. If you enable _all_ of the strict options, most of the errors seem to be null checks. For example, in `util.ts`, you call `done()` even though it is an optional parameter so could be undefined in reality.

It would probably be worth turning them on and seeing if there's any easy fixes you can do to keep at least things like `noUnusedLocals` and what not on.

Here's what I tried it with:

```
    "alwaysStrict": true,
    "noFallthroughCasesInSwitch": true,
    "noImplicitAny": true,
    "noImplicitReturns": true,
    "noImplicitThis": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "strict": true,
    "strictFunctionTypes": true,
    "strictNullChecks": true,
    "strictPropertyInitialization": true,
```